package.json

```diff
@@ -61,10 +62,5 @@
     "prettier": "^3.3.3",
     "storybook": "^8.3.4"
   },
-  "resolutions": {
-    "**/recursive-readdir/**/minimatch": "^3.0.5",
-    "**/css-select/**/nth-check": "^2.1.1",
-    "**/react-scripts/**/semver": "^7.5.2",
-    "**/react-scripts/**/postcss": "^8.4.31"
-  }
```

Comment:
Those were added a few months ago because we had outdated dependencies causing security issues (mostly related to create-react-app not being maintained anymore). 
I'd rather keep them and take the time to check separately if we still need them 👍

Reply:
If I add them to the `package.json`, running `yarn install` will remove them again.

By looking at the `yarn.lock` file, it appears those resolutions are already satisfied though 😄